site/src/docs/components/footer/customisation.mdx

```diff
@@ -46,7 +46,12 @@ import { Footer } from 'hds-react';
       <Link label="Item" />
       <Link label="Item" />
     </Footer.Navigation>
-    <Footer.Base copyrightHolder="Copyright" copyrightText="All rights reserved" backToTopLabel="Back to top">
+    <Footer.Base
+      copyrightHolder="Copyright"
+      copyrightText="All rights reserved"
+      backToTopLabel="Back to top"
+      logo={<Logo src={logoFi} size="medium" />}
+    >
       <Link label="Link" />
       <Link label="Link" />
       <Link label="Link" />
```

Comment:
Links are not showing, they should be `Footer.Link`. This was my bad from my previous renaming components PR.